### PR TITLE
reduce batman dependency in status page

### PR DESCRIPTION
**Pull request by christf**
this requiries gluon.mesh

### CODE REVIEW

**Comment by christf** [comment]
as tcatm requested, raising this for the master branch instead of babel.

File: package/gluon-status-page-api/files/lib/gluon/status-page/www/cgi-bin/dyn/neighbours-nodeinfo [hunk 1]
```diff
@@ -8,6 +8,7 @@ badrequest() {
 
 echo 'Access-Control-Allow-Origin: *'
 
-batctl if | cut -d: -f1 | grep -qxF "$QUERY_STRING" || badrequest
+#batctl if | cut -d: -f1 | grep -qxF "$QUERY_STRING" || badrequest
```
Comment (rotanid) [inline package/gluon-status-page-api/files/lib/gluon/status-page/www/cgi-bin/dyn/neighbours-nodeinfo]: why leaving the old commands there? if we need them again, we could look them up in git history.
(my comment also applies to the other file changes in your commit)

**Comment by christf** [comment]
right.

**Comment by christf** [comment]
- [ ] re-implement shell scripts in commits in lua

**Comment by tcatm** [comment]
Does this output an empty line after "Status: 400 Bad Request"?

**Comment by christf** [comment]
yes it does.

**Comment by NeoRaider** [comment]
The "blocked" label marks issues/PRs that depend on other issues/PRs (in this case the gluon.mesh support in #747).

**Comment by rotanid** [comment]
removing the blocked label as the dependency was merged in early september.
needs a rebase now, if you still want that change, @christf

**Comment by christf** [comment]
@rotanid this needs rework. gluon.mesh is not available any more instead interfaces from uci should be used having proto = gluon_mesh there is already a reference implementation in the babel init script which should be used here as well. better yet it should be put in a central place and be called from both of these scripts.

**Comment by christf** [comment]
@lemoer  referencinghim so he is aware too.

**Comment by christf** [comment]
this is superseeded by the work in #934

**Pull request closed without merging**